haystack/pipelines/utils.py

```diff
@@ -250,20 +254,24 @@ def print_eval_report(
         n_wrong_examples=n_wrong_examples,
         document_scope=document_scope,
         answer_scope=answer_scope,
+        wrong_examples_filter=wrong_examples_filter,
+        max_characters_per_wrong_examples_report=max_characters_per_wrong_examples_report,
     )
 
     print(f"{pipeline_overview}\n" f"{wrong_examples_report}")
 
 
-def _format_document_answer(document_or_answer: dict):
-    return "\n \t".join(f"{name}: {value}" for name, value in document_or_answer.items())
+def _format_document_answer(document_or_answer: dict, field_filter: List[str] = None):
+    if field_filter is None or len(field_filter) == 0:
+        field_filter = document_or_answer.keys()  # type: ignore
```

Comment:
Can we get around the type ignore somehow? Maybe converting it in a `Set[str]` variable before filtering...

Reply:
Ok, I'll see what I can do.